src/AnimateUtil/AnimateUtil.js

```diff
@@ -14,7 +14,7 @@ class AnimateUtil {
    * in the end with given `duration` in ms, using the given style.
    *
    * @param {ol.Map} map An OlMap.
-   * @param {ol.layer.Vector} layer A vector layer which feature should be moved.
+   * @param {ol.layer.Vector|boolean} layer A vector layer to receive an postrender event or false.
```

Comment:
```suggestion
   * @param {ol.layer.Vector|null} layer A vector layer to receive an postrender event or null.
```

Wouldn't `null` make more sense? Instead of false?

Reply:
I guess this is a matter of taste. Personally I like the `boolean` better, because it looks more explicit.